### PR TITLE
add deprecation notice to socialbakers and emplifi connectors

### PR DESCRIPTION
**Pull request by alesmenzel**
Link to Emplifi (formerly Socialbakers) docs: https://docs.emplifi.io/platform/latest/home/how-to-set-up-power-bi-connector.

### CODE REVIEW

**Comment by alesmenzel** [comment]
@microsoft-github-policy-service agree company="Emplifi"

**Comment by alesmenzel** [comment]
Hi, is there anything else required to merge this PR?

**Comment by DougKlopfenstein** [comment]
@alesmenzel - "(Deprecated)" should also be added to the table of contents entries for both the Emplify Metrics and the Socialbaker Metrics connectors. The table of contents file is toc.yml.

In addition, "(Deprecated)" should be added to the list of all connectors instance for both the Emplify Metrics and the Socialbaker Metrics connectors. The list of all connectors file is connectors/index.md. In this file, you should also change all check marks (yes.png) to "X" (no.png) to indicate the deprecation.

**Review by DougKlopfenstein — APPROVED** [review]

**Comment by DougKlopfenstein** [comment]
#sign-off

File: powerquery-docs/connectors/socialbakers.md [hunk 1]
```diff
@@ -7,19 +7,21 @@ ms.date: 3/14/2024
 ms.author: dougklo
 ---
 
-# Socialbakers (Beta)
+# Socialbakers (Deprecated)
+
+Sunset of Socialbakers API Connector is scheduled for August 1st, 2024. Please make sure to make required changes beforehand. Contact your Emplifi representative for more info.
```
Comment (Court72) [inline powerquery-docs/connectors/socialbakers.md:12]:
```suggestion
Sunset of Socialbakers API Connector is scheduled for August 1, 2024. Please make sure to make the required changes beforehand. Contact your Emplifi representative for more info.
```

**Pull request merged**